src/stacked_borrows.rs

```diff
@@ -362,6 +377,42 @@ impl<'tcx> Stack {
         Ok(())
     }
 
+    /// Inserts an `SbTag::Unknown` above the specified tag. This represents
+    /// exposing a pointer via an ptrtoint cast.
```

Comment:
Exposing shouldn't change stacks. It doesn't even know which stacks it would possibly have to change.